### DISCUSSION TITLE
maddy 0.5.4

### DISCUSSION THREAD

**Discussion by foxcpp** [emersion/maddy#464] [General]
## Bugfixes

* auth/pam: Fix double-free crash (GH #272)
* [security] auth/pam: Check for account/password expiry (GHSA-6cp7-g972-w9m9)
* endpoint/smtp: Fix incoming message log record having no msg_id

<hr /><em>This discussion was created from the release <a href='https://github.com/foxcpp/maddy/releases/tag/v0.5.4'>maddy 0.5.4</a>.</em>

**Comment by arisudesu** [comment]
You forgot to push the branch containing the tagged commit. Please do it as well.

**Reply by foxcpp** [reply]
Woops, fixed.
